### DISCUSSION TITLE
Webhooks schema

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#6248] [Feature Requests]
I think it would be awesome if we could create a custom post-schema for the webhooks.
This would allow to send a post-request directly to a third-party service to sync data with other services (e.g post events to the google calendar API). 

The configuration could be make comparably to the permissions editor as a JSON field.

**Comment by azrikahar** [comment]
I believe this is technically achievable now via [Flows Webhook/Request URL Operation](https://docs.directus.io/app/flows/operations.html#webhook-request-url), so I'll mark this as resolved. Happy to amend this if deemed otherwise.
